src/main/java/tech/jhipster/lite/generator/client/tools/cypressmergecoverage/domain/CypressMergeCoverageModuleFactory.java

```diff
@@ -0,0 +1,90 @@
+package tech.jhipster.lite.generator.client.tools.cypressmergecoverage.domain;
+
+import static tech.jhipster.lite.module.domain.JHipsterModule.fileStart;
+import static tech.jhipster.lite.module.domain.JHipsterModule.from;
+import static tech.jhipster.lite.module.domain.JHipsterModule.lineAfterRegex;
+import static tech.jhipster.lite.module.domain.JHipsterModule.packageName;
+import static tech.jhipster.lite.module.domain.JHipsterModule.path;
+import static tech.jhipster.lite.module.domain.JHipsterModule.regex;
+import static tech.jhipster.lite.module.domain.JHipsterModule.scriptCommand;
+import static tech.jhipster.lite.module.domain.JHipsterModule.scriptKey;
+import static tech.jhipster.lite.module.domain.JHipsterModule.text;
+import static tech.jhipster.lite.module.domain.JHipsterModule.to;
+import static tech.jhipster.lite.module.domain.npm.JHLiteNpmVersionSource.COMMON;
+
+import tech.jhipster.lite.module.domain.JHipsterModule;
+import tech.jhipster.lite.module.domain.file.JHipsterSource;
+import tech.jhipster.lite.module.domain.properties.JHipsterModuleProperties;
+
+public class CypressMergeCoverageModuleFactory {
+
+  private static final JHipsterSource SOURCE = from("client/tools/cypressmergecoverage");
+  private static final String CYPRESS_COMPONENT_TESTS = "src/test/webapp/component";
+
+  public JHipsterModule buildCypressMergeCoverage(JHipsterModuleProperties properties) {
+    return JHipsterModule.moduleBuilder(properties)
```

Comment:
Usually we format manually the builder invocation, this allow to use a more readable indent. Take a look to other factories, e.g. `CypressModuleFactory`
```suggestion
    //@formatter:off
    return JHipsterModule.moduleBuilder(properties)
```